Source/WebGPU/WGSL/AST/ASTStringDumper.cpp

```diff
@@ -307,6 +307,22 @@ void StringDumper::visit(CompoundStatement& block)
     m_out.print("}\n");
 }
 
+void StringDumper::visit(IfStatement& statement)
+{
+    m_out.print(m_indent, "if ");
+    visit(statement.test());
+    m_out.print("\n");
+    visit(statement.trueBody());
```

Comment:
nit: will this include curly braces? if so there's probably no need to add a line break above